doc/api/buffer.md

```diff
@@ -2606,8 +2606,9 @@ Returns an un-pooled `Buffer`.
 
 In order to avoid the garbage collection overhead of creating many individually
 allocated `Buffer` instances, by default allocations under 4KB are sliced from a
-single larger allocated object. This approach improves both performance and memory
-usage since v8 does not need to track and cleanup as many `Persistent` objects.
+single larger allocated object. This approach improves both performance and
+memory usage since V8 does not need to track and cleanup as many `Persistent`
+objects.
```

Comment:
This sentence is both too implementation-specific and factually wrong (there are no `Persistent` handles involved for pooled buffers), I’d just remove it (or shorten to `… single larger allocated object to improve performance.`).

Reply:
Thanks. Removed it.